lib/Init.vala

```diff
@@ -4,13 +4,20 @@
  */
 
 namespace Granite {
-    static bool initialized = false;
+    private static bool initialized = false;
 
-    internal void init () {
+    /*
+     * Initializes Granite.
+     * If Granite has already been initialized, the function will return.
+     * Makes sure translations and types for Granite are set up properly.
+     */
+    public void init () {
         if (initialized) {
             return;
         }
 
+        typeof (Granite.Settings).ensure ();
```

Comment:
You're already inside Granite namespace, so jut
```suggestion
        typeof (Settings).ensure ();
```
would have worked. Might be a matter of code style tho, not sure.

Reply:
Yeah I just want to make sure we disambiguate between GLib.Settings just in case!